Sources/TuistKit/Generator/WorkspaceGenerator.swift

```diff
@@ -59,22 +64,59 @@ final class WorkspaceGenerator: WorkspaceGenerating {
         let workspacePath = workspaceRootPath.appending(component: workspaceName)
         let workspaceData = XCWorkspaceData(children: [])
         let workspace = XCWorkspace(data: workspaceData)
+        
+        /*-----------------------------------------------------------------------------------------
+         // MARK: - Manifest Group
+         -----------------------------------------------------------------------------------------*/
 
+        let manifestFiles: [XCWorkspaceDataElement] = [ Manifest.workspace, Manifest.setup ]
```

Comment:
Not a big fan of operators but I have to say this one reads way better. Let's give it a try.

Reply:
was the motivation to support mapping KeyPaths in a single step?

Some alternatives:

a mix and match:

```swift
let manifestFiles: [XCWorkspaceDataElement] = [Manifest.workspace, Manifest.setup]
            .lazy
            .map { path.appending(component: $0.fileName) }
            .filter(fileHandler.exists)
            .map(\.workspaceFileElement)
```

or with an extra step if you prefer avoiding writing inline closures `{ }` 😛

```swift
let manifestFiles: [XCWorkspaceDataElement] = [Manifest.workspace, Manifest.setup]
            .lazy
            .map(\.fileName)
            .map(path.appending(component:))
            .filter(fileHandler.exists)
            .map(\.workspaceFileElement)
```

Reply:
One issue I spotted - for the manifests, we end up using their absolute paths in the generated workspace :( - this would prevent checking in generated projects / workspaces 

We'll need to map back to relative path 

```swift
            .filter(fileHandler.exists)
            .map { $0.relative(to: path) }
            .map(\.workspaceFileElement)
```

Reply:
Good catch, I forgot that we should be using relative.

Reply:
Thanks for the suggestions @kwridan - I like the implementation.

Reply:
I removed the operator and used the named function instead!